doc/nrf/gs_testing.rst

```diff
@@ -150,13 +150,6 @@ When using an ``_ns`` build target, by default you can only debug firmware in th
 
 To debug firmware running in the secure part, you need to build Trusted Firmware-M with debug symbols enabled and load the symbols during the debugging session.
 To build Trusted Firmware-M with debug symbols, set the :kconfig:option:`CONFIG_TFM_CMAKE_BUILD_TYPE_RELWITHDEBINFO` Kconfig option.
-To load the Trusted Firmware-M debug symbols in the GDB tool, run the following command and answer ``y`` if prompted by the tool:
-
-.. code-block:: console
-
-   (gdb) add-symbol-file build/tfm/bin/tfm_s.elf
-
-For more information about the GDB tool, see `Debug tools`_.
```

Comment:
We have nothing similar valid for nRF Debug?

Reply:
No, nRF Debug does this automatically according to @trond-snekvik.